### ISSUE TITLE
"GPS" tile broken on Android 9+

### ISSUE THREAD

**Issue by ztNFny** [GravityBox/GravityBox#2034]
The GPS quick settings tile modifies settings secure location_mode, but that one is deprecated (https://developer.android.com/reference/android/provider/Settings.Secure.html#LOCATION_MODE) and not used at all anymore from Android 9 on.
The old 3 location modes (device only (gps), battery saving (network), high accuracy (gps + network)) were replaced by a binary toggle in settings. Now there's location_mode = 0 (off) & 1/2/3 (on) with 3 being the "normal" on.
The sources are controlled by settings get secure location_providers_allowed with values "", "gps", "network", "gps,network"

The GPS tile should thus be changed to perform
settings put secure location_providers_allowed +gps
settings put secure location_providers_allowed -gps

**Comment by C3C0** [comment]
Yes, I know about this and need to find time to adjust it for Pie and Q. Thanks for analysis, though.
